flask_dance/__init__.py

```diff
@@ -3,4 +3,4 @@
 
 from .consumer import OAuth1ConsumerBlueprint, OAuth2ConsumerBlueprint
 
-__version__ = "1.3.0"
+__version__ = "1.3.1"
```

Comment:
Please don’t touch the version number. Put the changelog entries in the “unreleased” section.